archetypes/quickstart-se/src/main/resources/Dockerfile.jlink.mustache

```diff
@@ -20,6 +20,6 @@ RUN echo "done!"
 
 FROM debian:stretch-slim
 WORKDIR /helidon
-COPY --from=build /helidon/target/{{artifactId}} ./
+COPY --from=build /helidon/target/{{artifactId}}-jri ./
```

Comment:
This also needs to be fixed in the quickstart-**mp**.

Reply:
Thanks for pointing that out !